### PR TITLE
Finish with rename

### PR DESCRIPTION
**Pull request by prachigauriar**

### CODE REVIEW

File: Task/Workflow/TSKWorkflow+TaskInterface.h [hunk 1]
```diff
@@ -40,14 +40,14 @@ NS_ASSUME_NONNULL_BEGIN
...
-- (void)subtask:(TSKTask *)task didFinishWithResult:(nullable id)result;
+- (void)subtask:(TSKTask *)task didFinishWithResult:(nullable id)result NS_SWIFT_NAME(subtask(_:didFinishWith:));
 
 /*!
  @abstract Indicates to the workflow that the specified task failed.
  @param task The task that failed. May not be nil.
  @param error The error that caused the task to fail.
  */
-- (void)subtask:(TSKTask *)task didFailWithError:(nullable NSError *)error NS_SWIFT_NAME(subtask(_:didFailwithError:));
+- (void)subtask:(TSKTask *)task didFailWithError:(nullable NSError *)error NS_SWIFT_NAME(subtask(_:didFailWith:));;
```
Comment (jnjosh) [inline Task/Workflow/TSKWorkflow+TaskInterface.h]: You are a little semicolon happy here.

**Review by jnjosh — APPROVED** [review]
👍

**Review by macdrevx — APPROVED** [review]

File: Task/Workflow/TSKWorkflow+TaskInterface.h (hunk 1 above)
Comment (prachigauriar) [inline Task/Workflow/TSKWorkflow+TaskInterface.h]: Indeedy doo.

Comment (prachigauriar) [inline Task/Workflow/TSKWorkflow+TaskInterface.h]: Will fix.

**Pull request merged**